library/src/main/java/com/novoda/downloadmanager/lib/FileDownloadInfo.java

```diff
@@ -15,14 +12,12 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.List;
-import java.util.concurrent.ExecutorService;
 
 /**
  * Stores information about an individual download.
  */
 class FileDownloadInfo {
 
-    public static final String EXTRA_EXTRA = "com.novoda.download.lib.KEY_INTENT_EXTRA";
```

Comment:
The value is still the same and that constant wasn't publically accessible!
Should be a ok
On 10 Jul 2015 08:22, "Paul Blundell" <notifications@github.com> wrote:

> In
> library/src/main/java/com/novoda/downloadmanager/lib/FileDownloadInfo.java
> <https://github.com/novoda/download-manager/pull/92#discussion_r34332571>:
>
> >
> >  /**
> >   * Stores information about an individual download.
> >   */
> >  class FileDownloadInfo {
> >
> > -    public static final String EXTRA_EXTRA = "com.novoda.download.lib.KEY_INTENT_EXTRA";
>
> fyi this is a public api breaking change so remember to write that in the
> changelog
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/download-manager/pull/92/files#r34332571>.
>

-- 
Disclaimer: The information in this e-mail and any attachments is the 
property of Novoda Ltd and is confidential and may be legally privileged. 
This e-mail is intended solely for the person or organisation to which it 
is addressed. Any disclosure, copying or other use of the information by 
any person or organisation who is not the intended recipient is strictly 
prohibited and may be unlawful. If you have received this e-mail in error, 
please inform the sender immediately and delete/destroy this e-mail and any 
copies of it. Novoda Ltd has taken reasonable precautions to minimise the 
risk of any software viruses which may damage your systems, but we advise 
that you take the necessary steps to ensure that no virus contamination is 
suffered. Novoda Ltd does not accept any liability for any loss or damage 
caused by the transmission of any virus.

Novoda Ltd, Company No: 347444, Registered in Scotland Registered 
Office: C/O Alexander Sloan, 38 Cadogan Street, Glasgow, G2 7HF, Scotland.  VAT 
Registration Number  GB 984 2525 93